sympy/polys/polyroots.py

```diff
@@ -842,6 +842,7 @@ def roots(f, *gens,
 
     >>> from sympy import Poly, roots
     >>> from sympy.abc import x, y
+    >>> from sympy.polys import degree
```

Comment:
This should just be `from sympy import degree`. The docs shouldn't encourage users to import from submodules.

Reply:
Actually it should just be added to the import above.